scripts/update_bookings_during_quarantine.py

```diff
@@ -0,0 +1,25 @@
+from datetime import datetime
+from typing import List
+from models import Booking, Stock, Payment
+from repository import repository
+
+
+def update_booking_status_for_events_happening_during_quarantine():
```

Comment:
```suggestion
def cancel_bookings_status_for_events_happening_during_quarantine():
```
+ le nom du fichier.

Reply:
J'aurais même mis cancel_bookings_for_events_happening_during_quarantine mais c'est un détail :)